src/assets/i18n/en.json5

```diff
@@ -1620,6 +1620,12 @@
 
   "error.validation.groupExists": "This group already exists",
 
+  "error.validation.metadata.namespace.invalid-pattern": "This field cannot contain dots, please use the Element & Qualifier fields instead",
+
+  "error.validation.metadata.element.invalid-pattern": "This field cannot contain dots, please use the Qualifier field instead",
+
+  "error.validation.metadata.qualifier.invalid-pattern": "This field cannot contain dots",
+
```

Comment:
Please update it according to the final excluded chars, This field cannot contain inner spaces, dots or underscore _, please ...